src/DevBetterWeb.Web/Pages/User/UserProfileUpdateModel.cs

```diff
@@ -10,28 +14,32 @@ namespace DevBetterWeb.Web.Pages.User
 {
     public class UserProfileUpdateModel
     {
+
         [Required]
         public string? FirstName { get; set; }
         [Required]
         public string? LastName { get; set; }
         public string? Address { get; set; }
-        [ValidUrl]
+        //[ValidUrl]
```

Comment:
Why are we changing ValidUrl? Did it break somehow?

Reply:
it was broken at one point, while I had a work-in-progress. Should be ok to change it back now. I will try

Reply:
it was broken because of some of the changes we made with nullability

Reply:
so essentially, in ValidUrlAttribute.cs, the overriding IsValid method is mad about the nullability of its parameter value, because we enabled nullability in the project. Can I suppress that error for that line of code?